covid_act_now/tests/conftest.py

```diff
@@ -20,12 +19,26 @@ def run_as_module():
 
     with mock_s3():
         # Create the fake bucket we will be using
-        params = read_params()
+        params = {
+            "common": {
+                "export_dir": "./receiving"
+            },
+            "indicator": {
+                "parquet_url": "./test_data/small_CAN_data.parquet"
+            },
+            "archive": {
+                "cache_dir": "./cache",
+                "bucket_name": "test-bucket",
+                "aws_credentials": {
+                    "aws_access_key_id": "{{ delphi_aws_access_key_id }}",
+                    "aws_secret_access_key": "{{ delphi_aws_secret_access_key }}"
+                }
+            }
+        }
         aws_credentials = params["archive"]["aws_credentials"]
```

Comment:
Since we aren't actually doing any archive testing, it might make sense here just to totally remove the whole s3 mocking and the archive parts of the params.

Reply:
:+1: removed